.github/workflows/test-unit.yml

```diff
@@ -7,6 +7,7 @@ env:
   GO_VERSION: 1.22.2
   TEST_PARALLELISM: 12
   TEST_PARALLELISM_PKG: 6
+  TEST_COVERAGE: false
```

Comment:
This has to be set to cache the test results into `go build testcache`. I will finish the CI when and how to execute the `TEST_COVERAGE:true` part.